src/index/blockfilterindex.cpp

```diff
@@ -222,9 +202,9 @@ size_t BlockFilterIndex::WriteFilterToDisk(FlatFilePos& pos, const BlockFilter&
 {
     assert(filter.GetFilterType() == GetFilterType());
 
-    size_t data_size =
+    uint64_t data_size{
```

Comment:
This is used later
https://github.com/bitcoin/bitcoin/blob/fab75c2d0332fdb25030abb441363478cd486dc5/src/index/blockfilterindex.cpp#L236
for a method that still expects `size_t`
https://github.com/bitcoin/bitcoin/blob/c10e382d2a3b76b70ebb8a4eb5cd99fc9f14d702/src/flatfile.cpp#L57

Reply:
(see above)